butterknife-compiler/src/main/java/butterknife/compiler/BindingSet.java

```diff
@@ -82,13 +88,16 @@ private BindingSet(TypeName targetTypeName, ClassName bindingClassName, boolean
     this.parentBinding = parentBinding;
   }
 
-  JavaFile brewJava(int sdk, boolean debuggable) {
-    return JavaFile.builder(bindingClassName.packageName(), createType(sdk, debuggable))
+  JavaFile brewJava(int sdk, boolean debuggable, boolean useAndroidX) {
+    return JavaFile.builder(bindingClassName.packageName(), createType(
```

Comment:
nit: can we pull the `createType` call into a local. I really don't like the wrapping here.

Reply:
34b4a43